popularity/tool/build.dart

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env dart --checked
```

Comment:
Maybe update the `README.md` file and update it to include instructions for re-building the generated code here.